packages/use-sync-external-store/src/useSyncExternalStore.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export function useSyncExternalStore() {
```

Comment:
Will this be a default export?

Reply:
Idk I just did it this way because that's how useSubscription does it. Can decide later.